keyboards/kc60se/kc60se.h

```diff
@@ -77,5 +63,4 @@ inline void kc60se_caps_led_off(void) { DDRB &= ~(1<<2); PORTB &= ~(1<<2); }
     { K40, K41, K42, ___, ___, K45, ___, ___, ___, K49, K4A, K4B, K4C, K4D }  \
 }
 
-
-#endif
+// clang-format on
```

Comment:
As you've renamed the `LAYOUT` macro, you should include the following define to maintain backwards compatibility for users with keymaps out-of-tree.

```suggestion
#define LAYOUT LAYOUT_all
// clang-format on
```